app/multitenant/dynamo_collector.go

```diff
@@ -87,6 +97,11 @@ type DynamoDBCollector interface {
 	CreateTables() error
 }
 
+type watchKey struct {
```

Comment:
I'd really welcome a brief comment explaining what this is & what's it's used for.